shallowblue.py

```diff
@@ -23,9 +23,16 @@
 
 # Hyperparameters
 SIZE = 20
-SIZE_LESS = 19
-DIAMOND_DENSITY = .01
-COAL_DENSITY = .03
+
+SIZE_POOL = SIZE - 1  # water stretches size-1 in each direction
+
+WATER_DEPTH = 5  # don't set to above 20 or else it might break and the server might need to be restarted
+
+AGENT_START_Y = WATER_DEPTH + 1  # agent always starts 1 above the water (to prevent excessive falling or other stuff)
+
+DIAMOND_DENSITY = .1
+COAL_DENSITY = .2
```

Comment:
These densities may be a bit too high? I think lower densities and more training may make our problem a bit more interesting.